recipes/conda-env/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.4.5" %}
```

Comment:
[2.5.1](https://github.com/conda/conda-env/releases) is the latest if we want to update this.